Sources/XcodeEdit/PBXObject.swift

```diff
@@ -269,6 +272,19 @@ public class XCRemoteSwiftPackageReference : PBXProjectItem {
   }
 }
 
+public class XCLocalSwiftPackageReference : XCSwiftPackageReference {
+  public let relativePath: Path
+  let pathString: String
+
+  public required init(id: Guid, fields: Fields, allObjects: AllObjects) throws {
+      pathString = try fields.string("relativePath")
+      self.relativePath = .relativeTo(.sourceRoot, pathString)
```

Comment:
pathString would be relative path (`../some/dir`), but I'm not 100% sure whether this is the intended use for `.relativeTo`

Reply:
This works in my testing.